src/gen-ts.ts

```diff
@@ -136,6 +139,21 @@ function analyzerToAst(
   return tsDocs;
 }
 
+/**
+ * Analyzer always returns fully specified URLs with a protocol and an absolute
+ * path (e.g. "file:/foo/bar"). Return just the file path, relative to our
+ * project root.
+ */
+function analyzerUrlToRelativePath(
+    analyzerUrl: string, rootDir: string): string {
+  const parsed = url.parse(analyzerUrl);
+  if (parsed.protocol !== 'file:' || parsed.path === undefined) {
```

Comment:
Ensure that parsed.host is empty too. `file://example.com/path/in/example` is a legit url

Reply:
Done. Also realized that parse returns empty string, not undefined (even though the type possibly returns undefined).